sources/packages/backend/apps/api/src/services/form/form.service.ts

```diff
@@ -43,13 +53,32 @@ export class FormService {
   }
 
   /**
-   * Lists form definitions that contains the tag 'common'.
+   * Lists form definitions that contains the tag 'common' ordered by title.
    */
-  async list() {
+  async list(): Promise<FormDefinition[]> {
+    const authHeader = await this.createAuthHeader();
     const content = await this.httpService.axiosRef.get(
-      `${this.config.formsUrl}/form?type=form&tags=common`,
+      `${this.config.formsUrl}/form?tags=common&limit=${FORMIO_PAGE_LIMIT}&sort=${FORMIO_LIST_SORT_FIELD}`,
```

Comment:
Asking for my understanding why was the param `tags=common` used here?

Reply:
"Common" is a tag added to all the forms created for the application. It is used just to filter some out-of-the-box forms that come with form.io server.
It was a simple way to keep using what was already in place. The tags can be explored and organized better in the future.